### PR TITLE
DOCS-1905 API Tests Network Layer Section and Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Inserted a tile partial to visualize the API network layer test options.

### Motivation

DOCS-1905

### Preview

https://docs-staging.datadoghq.com/alai97/synthetics-api-tests-network-layers/synthetics/api_tests/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: layouts/partials/synthetics/network-layers.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{{ $dot := . }}
+<div class="logs-containers">
+    <div class="container cards-dd">
+        <div class="card-deck">
+            <a class="card" href="/api_tests/http_tests">
```
Comment (bgdeutsch) [inline layouts/partials/synthetics/network-layers.html:5]: this link is 404'ing for me - maybe missing the `/synthetics/` ?

Comment (alai97) [reply layouts/partials/synthetics/network-layers.html:5]: Great catch!

```suggestion
            <a class="card" href="/synthetics/api_tests/http_tests">
```

**Review by bgdeutsch — APPROVED** [review]

File: content/en/synthetics/api_tests/_index.md [hunk 2]
```diff
@@ -25,33 +25,26 @@ further_reading:
...
+
+{{< partial name="synthetics/network-layers.html" >}}
 
 {{< img src="synthetics/api_tests/api_test.mp4" alt="API Test subtypes" video="true"  width="100%" >}}
 
-If your service starts answering more slowly, or in an unexpected way (for example, unexpected response body, wrong A record, etc.), your test can [**alert your team**][6], [**block your CI pipeline**][7], or even [**roll back the faulty deployment**][7].
+If your service starts answering slower or in an unexpected way (such as an unexpected response body or wrong A record), your test can [alert your team][1], [block your CI pipeline][2], and [roll back the faulty deployment][2].
 
-API tests can run from Datadog [managed locations][8] and [private locations][9], allowing **full coverage of your systems**, both external and internal.
+API tests can run from Datadog [managed locations][3] and [private locations][4], allowing **internal and external coverage** of your systems.
```
Comment (sarina-dd) [inline content/en/synthetics/api_tests/_index.md:38]:
```suggestion
API tests run from Datadog [managed locations][3] or [private locations][4], allowing **internal and external coverage** of your systems.
```

File: content/en/synthetics/api_tests/_index.md [hunk 3]
```diff
@@ -25,33 +25,26 @@ further_reading:
...
 
-**Note**: API tests are single requests executed against your services. If you want to monitor sophisticated business transactions at the API level or endpoints that require authentication, you can also chain your requests using [Multistep API tests][10].
+**Note**: API tests are single requests executed against your services. If you want to monitor sophisticated business transactions at the API level or endpoints that require authentication, you can chain your requests with [multistep API tests][5].
```
Comment (sarina-dd) [inline content/en/synthetics/api_tests/_index.md:40]:
```suggestion
**Note**: API tests are single requests executed against your services. If you want to monitor sophisticated business transactions at the API level or endpoints that require authentication, chain your requests with [multistep API tests][5].
```

**Review by sarina-dd — APPROVED** [review]

**Pull request merged**